dvc/command/get.py

```diff
@@ -52,6 +56,6 @@ def add_parser(subparsers, parent_parser):
         help="Destination path to copy/download files to.",
```

Comment:
```suggestion
        help="Destination path to download files to",
```
since we're no longer using the term "copy" in these texts

Reply:
Not sure why GH thinks my suggestion removes a space character. It doesn't. That's weird. Oh well...